CIScripts/SpawnVM.ps1

```diff
@@ -70,7 +73,7 @@ function New-TestbedVMs {
             Start-Sleep -s 30
```

Comment:
this 30 seconds sleep time should be included in the calculation of number of retries.